pages/docs/learn/inngest-functions.mdx

```diff
@@ -0,0 +1,104 @@
+
+Let's have a look at each of this function's components.
+
+<Callout>
+💡 You can test Inngest functions using standard tooling such as Jest or Mocha. To do so, export the job code and run standard unit tests.
+</Callout>
+
+### Config
+
+The first parameter of the `createFunction` method specifies Inngest function's configuration. In the above example, the `id` is specified, which will be used to identify the function in the [Inngest Dev Server](/docs/local-development):
+
+<img
+  src="/assets/docs/local-development/dev-server-functions.png"
+  width="400" height={400/2762*1868} quality="95"
+  className="rounded-md"
+  alt="Screenshot of the Inngest Dev Server interface showing three functions listed under the 'Functions' tab. The functions are: 'store-events,' 'Generate monthly report,' and 'Customer Onboarding,' each with their respective triggers and App URLs."
+/>
+
+You can also provide other [configuration options](/docs/reference/functions/create#configuration), such as `concurrency`, `throttle`, `debounce`, `rateLimit`, `priority`, `batchEvents`, or `idempotency` (preventing duplicate events from triggering the function within 24 hours). You can also specify how many times the function will retry, what callback function will run on failure, and when to cancel the function.
+
+### Trigger
+
+Inngest functions are designed to be triggered by events or crons (schedules). Events can come from various sources like HTTP requests, your own code, or external webhooks. When an event is received, it triggers a corresponding function to execute the tasks defined in the function handler (see the ["Handler" section](/docs/functions/overview#handler) below).
+
+Each function needs at least one trigger. However, you can also work with [multiple triggers](/docs/guides/multiple-triggers) to invoke your function whenever any of the events or crons is received.
+
+### Handler
+
+A "handler" is the core function that defines what should happen when the function is triggered. In the example above, the handler contains only one function called `copyAllImagesToS3`.
+
+Handler receives context, which includes the event data (like `event.data.imageURLs` in the example above), logging configuration, and tools for managing execution flow (such as `step.run`, `step.sleep()`, or `step.waitForEvent()`.)
+
+<Callout>
+   💡 If your handler contains numerous functions, it is a good practice to wrap each one in a step. In this way, you can manage complex state easier and if any task fails, it will be retried independently from others.
+</Callout>
+
+To summarize, the handler allows you to define detailed logic that should happen in response to the triggering event.
+
+## Kinds of Inngest functions
+
+### <a href="/docs/guides/background-jobs" className="flex items-center gap-4"><IconConcurrency size="2rem"/> Background functions</a> {{anchor: false}}
```

Comment:
**idea**: We could use the tiles/cards for these with maybe 2 columns? I'm happy either way, just thought it might be an option!